src/core/utils/string.cpp

```diff
@@ -106,6 +106,77 @@ bool Utils::String::naturalSort(QString left, QString right, bool &result)
     return false;
 }
 
+bool Utils::String::naturalLessThan(const QString &left, const QString &right)
+{
+    // Return value `false` indicates QString `right` should go before QString `left`, otherwise, after
+    const bool case_sensitive = false;
```

Comment:
Did you mean to also put an extra argument in the function? Otherwise, this is useless.
If you plan to keep it, rename it to "caseSensitive"